cmd/kubectl-gadget/collector.go

```diff
@@ -165,11 +167,22 @@ func processCollectorCmdRun(cmd *cobra.Command, args []string) {
 		}
 	}
 
-	utils.GenericTraceCommand("process-collector", &params, args, "Status", callback, nil)
+	config := &utils.TraceConfig{
+		GadgetName:       "process-collector",
+		Operation:        "start",
+		TraceOutputMode:  "Status",
+		TraceOutputState: "Completed",
+		CommonFlags:      &params,
+	}
+
+	err := utils.RunTraceAndPrintStatusOutput(config, callback)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%s\n", err)
```

Comment:
@mauriciovasquezbernal I agree we shouldn't return `0` but I am wondering what would be the difference between using `RunE` and returning `err` with respect to print `err` and call `os.Exit(1)`. If there is not, I suggest using `RunE`. What do you say?

Reply:
`RunE` prints the usage by default if an error happens (which is not right in this case). We can use `SilenceUsage` to avoid that but not sure if it's worthy now. I'd say that we'd need to make a general pass over all the code to handle all these cases in the case way after this PR is merged.